### PR TITLE
Android search keyboard issue

### LINKED ISSUE #199: Inside pages on mobile search input close android keyboard repeatedly
@danfox01 fyi.

### PR DESCRIPTION
**Pull request by martypowell**
Fixes #199 - The window resize event was funking with the heart of the search, causing the keyboard to open and then collapse immediately, making it impossible to search.

The fix was to check the width with the window width at the time the resize event is called, and only calling the re-position logic if the width is different.

This logic is currently stored in the red-inside templates in an html snippet and should be removed as part of the release.

### CODE REVIEW

File: js/internal-template-events.js [hunk 1]
```diff
@@ -1,15 +1,59 @@
...
+	var windowWidth;
+
+	function $getSearchContainer(width) {
+		return isMobile(width)
+			? $('#search-container')
+			: $('#internal-search-container');
+	}
+
+	function isMobile(width) {
+		var mediaWidth = 990;
```
Comment (mxsnyder) [inline js/internal-template-events.js]: This introduces a logical dependency on a specific breakpoint. Granted, those don't change often.

**Comment by mxsnyder** [comment]
What logic is causing the problem?

File: js/internal-template-events.js (hunk 1 above)
Comment (martypowell) [reply js/internal-template-events.js]: @mxsnyder this part of the logic was unchanged form the original code

**Comment by martypowell** [comment]
@mxsnyder the resize event was being called 4 times when the search input was "clicked". The reason for that I don't know. But there was no reason it should, so I cached the width of the window and checked that, to prevent multiple resize events because the window wasn't resizing.

**Review by mxsnyder — APPROVED** [review]

**Pull request merged**